client/blocks/gsuite-stats-nudge/index.js

```diff
@@ -1,145 +0,0 @@
-/**
- * External dependencies
- */
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { localize } from 'i18n-calypso';
-import PropTypes from 'prop-types';
-import Gridicon from 'components/gridicon';
-
-/**
- * Internal Dependencies
- */
-import { Button, Card } from '@automattic/components';
-import { emailManagement } from 'my-sites/email/paths';
-import isGSuiteStatsNudgeDismissed from 'state/selectors/is-gsuite-stats-nudge-dismissed';
-import QueryPreferences from 'components/data/query-preferences';
-import SectionHeader from 'components/section-header';
-import { dismissNudge } from './actions';
-import { enhanceWithSiteType, recordTracksEvent } from 'state/analytics/actions';
-import { withEnhancers } from 'state/utils';
-
-/**
- * Style dependencies
- */
-import './style.scss';
-
-class GSuiteStatsNudge extends Component {
-	static propTypes = {
-		domainSlug: PropTypes.string.isRequired,
-		isDismissed: PropTypes.bool.isRequired,
-		recordTracksEvent: PropTypes.func.isRequired,
-		siteId: PropTypes.number.isRequired,
-		siteSlug: PropTypes.string.isRequired,
-		translate: PropTypes.func.isRequired,
-	};
-
-	componentDidMount() {
-		this.recordView();
-	}
-
-	componentDidUpdate( prevProps ) {
-		if ( prevProps.siteId && this.props.siteId && this.props.siteId !== prevProps.siteId ) {
-			this.recordView();
-		}
-	}
-
-	recordView() {
-		if ( this.isVisible() ) {
-			this.props.recordTracksEvent( 'calypso_gsuite_stats_nudge_view' );
-		}
-	}
-
-	recordClick = ( eventName ) => {
-		this.props.recordTracksEvent( eventName );
-	};
-
-	onDismissClick = () => {
-		this.recordClick( 'calypso_gsuite_stats_nudge_dismiss_icon_click' );
-		this.props.dismissNudge();
-	};
-
-	onStartNowClick = () => {
-		this.recordClick( 'calypso_gsuite_stats_nudge_start_now_button_click' );
```

Comment:
A reminder to deprecate the events that were removed in Tracks once that pull request is merged.